### PR TITLE
Feature/openapi autoupdate code

### PR DESCRIPTION
**Pull request by kinga-tittl**
Included also:

- Secure endpoints available only for specified customer
- Set proper parameter types in openapi.yml

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -431,6 +458,34 @@
...
+                            <goal>generate</goal>
+                        </goals>
+                        <configuration>
+                            <generatorName>spring</generatorName>
+                            <inputSpec>${project.basedir}/src/test/resources/contracts/e-bank/openapi.yml</inputSpec>
+                            <configOptions>
+                                <sourceFolder>src/main/java</sourceFolder>
+                                <library>spring-boot</library>
+                                <dateLibrary>java8</dateLibrary>
+                                <java8>true</java8>
```
Comment (tompalka) [inline pom.xml:477]: Shouldn't we use Java 11?

File: src/main/java/com/example/ebank/models/Transaction.java [hunk 2]
```diff
@@ -1,75 +1,73 @@
...
 public class Transaction {
 
-    @Id
-    @GeneratedValue
-    private Long id;
-
-    @ManyToOne
-    @JoinColumn(name = "account_id", nullable = false)
-    private Account account;
+	@Id
```
Comment (tompalka) [inline src/main/java/com/example/ebank/models/Transaction.java:12]: Please use formatter on this file (indent with spaces, not tabs)

**Review by tompalka — APPROVED** [review]
After quick look, I have some minor comments only

File: pom.xml (hunk 1 above)
Comment (kinga-tittl) [reply pom.xml:477]: Actually I didn't notice in documentation java11, only java8, but I can check it

**Pull request merged**